builder/libvirt/config.go

```diff
@@ -118,6 +122,10 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		c.DomainName = fmt.Sprintf("packer-%s", xid.New())
 	}
 
+	if len(c.BootDevices) == 0 {
+		c.BootDevices[0] = "hd"
+	}
```

Comment:
Thank you for the catch. It's been many years since I touched golang, and I think I need to spend some time relearning. This one is a bit annoying though as if I had done the testing I had intended (but apparently forgot), I would have caught this error.
Making the change and doing doing more complete testing, and then will update.

Reply:
Thank you.
About the testing part, I'm also guilty as I've not yet included any automated testing in the pipeline. That's something I should put on my roadmap soon. But meanwhile, I appreciate you testing this changeset manually.